cordova-common/src/events.js

```diff
@@ -42,7 +42,11 @@ module.exports.forwardEventsTo = function (eventEmitter) {
     if (!(eventEmitter instanceof EventEmitter))
         throw new Error('Cordova events could be redirected to another EventEmitter instance only');
 
-    EVENTS_RECEIVER = eventEmitter;
+    // Skipping forwarding to self to avoid infinite recursion.
+    // This is the case when the modules are npm-linked.
+    if(this !== eventEmitter) {
```

Comment:
nit: please add space after `if`. Also could you please reference JIRA here so it'd be more clear why we're checking this